### PR TITLE
added screens and basic routing

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
added screens and basic routing

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
+import { Routes, Route } from "react-router-dom";
+import Landing from './components/Landing';
+import Home from './components/Home';
```
Comment (Pratik1005) [inline src/App.js:4]: Use barrel export to import components in one line

**Review by Pratik1005 — APPROVED** [review]
Rest looks good

File: src/App.js (hunk 1 above)
Comment (mohammadkhizerkhan) [reply src/App.js:4]: sure thanks

File: src/components/Landing.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { Hero, Logo } from "../assets";
+
+function Landing() {
+  return (
+    <div>
+      <main className="landing-main">
+        <Logo />
+        <Hero />
+        
+        <div class="main">
```
Comment (manismk) [inline src/components/Landing.jsx:12]: change to className

File: src/components/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+function Home() {
+  return (
+    <div>
+      <NavBar />
+      <div class="home-cont">
+        <aside class="aside-cont">
+          <ul class="list list-stack">
+            <li class="list-item">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
```
Comment (manismk) [inline src/components/Home.jsx:18]: change class to className in all the places in this file.

**Review by manismk — APPROVED** [review]
Rest looks good. Good naming convention.

File: src/components/Home.jsx (hunk 3 above)
Comment (mohammadkhizerkhan) [reply src/components/Home.jsx:18]: sure manis thanks

**Pull request merged**